app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -60,8 +60,14 @@ const getNativeResourceRoutingInfo = (
 export const ManagedResourceObject = memo(
   ({ application, resource, artifactVersionsByState, artifactDetails, depth }: IManagedResourceObjectProps) => {
     const { kind, displayName } = resource;
+
     const routingInfo = getNativeResourceRoutingInfo(resource) ?? { state: '', params: {} };
-    const route = useSref(routingInfo.state, routingInfo.params);
+    const routeProps = useSref(routingInfo.state, routingInfo.params);
+
+    const displayLink = getExperimentalDisplayLink(resource);
+    const displayLinkProps = displayLink && { href: displayLink, target: '_blank', rel: 'noopener noreferrer' };
+
+    const linkProps = routeProps.href ? routeProps : displayLinkProps;
```

Comment:
Apparently the `route` conditional on L91 below was always hitting the truthy branch, because `useSref` always returns an object. This was leading to resources that didn't have links showing as `<a>` tags without an href. I'm checking for `href` here now so that we show a plain text name (no link) when we have neither a native sref _or_ a display link from the plugin.